UnitsNet.Tests/CustomCode/FuelEfficiencyTests.cs

```diff
@@ -17,25 +17,22 @@
 // Licensed under MIT No Attribution, see LICENSE file at the root.
 // Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
 
-using System;
-using UnitsNet.Units;
 using Xunit;
 
 namespace UnitsNet.Tests.CustomCode
 {
     public class FuelEfficiencyTests : FuelEfficiencyTestsBase
     {
-        protected override bool SupportsSIUnitSystem => false;
-        protected override double KilometersPerLiterInOneLiterPer100Kilometers => 100;
-        protected override double LitersPer100KilometersInOneLiterPer100Kilometers => 1;
-        protected override double MilesPerUkGallonInOneLiterPer100Kilometers => 282.4809363;
-        protected override double MilesPerUsGallonInOneLiterPer100Kilometers => 235.2145833;
-        
-        [Theory(Skip = "Conversion from 0 km/L results in infinity")]
-        [MemberData(nameof(UnitTypes))]
-        public override void ToUnit_FromDefaultQuantity_ReturnsQuantityWithGivenUnit(FuelEfficiencyUnit unit)
+        protected override double MetersPerCubicMeterInOneKilometerPerLiter => 1e6;
+        protected override double KilometersPerLiterInOneKilometerPerLiter => 1;
+        protected override double LitersPer100KilometersInOneKilometerPerLiter => 100;
+        protected override double MilesPerUkGallonInOneKilometerPerLiter => 2.824809363318222;
+        protected override double MilesPerUsGallonInOneKilometerPerLiter => 2.352145833333333;
+
+        [Fact(Skip = "See about changing this to MetersPerCubicMeter")]
+        public override void BaseUnit_HasSIBase()
```

Comment:
Again, it's up to you to decide if we should un-`[Skip]` it or change the skip reason...

Reply:
Or if you want, there is another solution that doesn't require us having the `[Skip]` warning as a permanent feature: we could use a boolean property, maybe something like `SupportsSIUnitSystem` to skip the generation of these `not-gonna-be-fixed` type of tests.. 🤔

Reply:
If a unit doesn't make sense or is simply never used, then I don't think we should add it just to make some si unit system logic work, which arguably is a very niche part of UnitsNet and probably little used to begin with.

Reply:
Removed the `MeterPerCubicMeter` in https://github.com/angularsen/UnitsNet/pull/1489/commits/d7169c17ec184b7a73d0817c106dd55f94b43ba7

Reply:
Oh wait, it didn't remove it from the `Units.json`.. (i have to do it by hand)